.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
 src/content/docs @nf-core/docs
+src/content/events/*/bytesize*.md @nf-core/bytesize
```

Comment:
```suggestion
src/content/events/*/bytesize*.md @nf-core/bytesize @nf-core/outreach 
```

Reply:
@FranBonath we are trying to automate who gets tagged for review on changes to these files. Do you prefer a seaprate bytesize team, or is outreach fine?